.gitignore

```diff
@@ -0,0 +1,3 @@
+**/.idea
+*.iml
+**/*.swp
```

Comment:
Not sure what the general take on this is, but my opinion is that IDE specific things should not live in the .gitignore of a project. The place for that is the user's .gitignore. While I am writing that, I see that this is also not honored in the linkerd2 repo, so not sure whether we want to comply by that urban legend :) @alpeb WDYT?

Reply:
My personal opinion is it doesn't hurt to include all that in the repo's `.gitignore`. Not particularly attached to this opinion though ;-)